src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
The nested for loop here is used to check if there are any duplicate values in the object. This results in a time complexity of O(n^2), which is not efficient for large objects. Instead, you can check for duplicates while you are inverting the object in the second loop. If a key already exists in the inverted object, you can return null immediately.